### PR TITLE
SwiftLint 0.31

### PR DESCRIPTION
**Pull request by djbe**
Addresses one issue of #600 (incorrect ACL warnings in Swift 5).

Also removed some unneeded availability annotations:
- `@available(..., tvOS 1.0, ...)` doesn't do anything because the first version of `tvOS` was 9.0.
-  `@available(iOS 1.0, tvOS 1.0, watchOS 1.0, *)` just means "available" for those platforms, so we can skip it.

### CODE REVIEW

**Comment by SwiftGen-Eve** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should make sure that <code>Podfile.lock</code> and <code>Package.resolved</code> are changed in sync</td>
    </tr>
  </tbody>
</table>

Hey 👋 I'm Eve, the friendly bot watching over SwiftGen 🤖

Thanks a lot for your contribution!



---

Once you fix those tiny nitpickings above, we should be good to go! 🙌

ℹ️ _I will update this comment as you add new commits_

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: templates/xcassets/swift3.stencil [hunk 1]
```diff
@@ -10,7 +10,7 @@
 {% set imageAlias %}{{param.imageAliasName|default:"AssetImageTypeAlias"}}{% endset %}
 {% set forceNamespaces %}{{param.forceProvidesNamespaces|default:"false"}}{% endset %}
 {% set accessModifier %}{% if param.publicAccess %}public{% else %}internal{% endif %}{% endset %}
-#if os(OSX)
+#if os(macOS)
```
Comment (AliSoftware) [inline templates/xcassets/swift3.stencil:13]: So did you check that the name macOS was available as soon as 3.0 and not just 3.1 then (as Brent wasn't totally sure in Slack)?

**Review by AliSoftware — APPROVED** [review]

File: templates/xcassets/swift3.stencil (hunk 1 above)
Comment (djbe) [reply templates/xcassets/swift3.stencil:13]: Jep, it's in the 3.0 branch:
https://github.com/apple/swift/blob/swift-3.0-branch/lib/AST/PlatformKind.cpp#L54

Comment (djbe) [reply templates/xcassets/swift3.stencil:13]: And the 3.0 release: https://github.com/apple/swift/blob/swift-3.0-RELEASE/lib/AST/PlatformKind.cpp#L54

**Pull request merged**